instrumentation/github.com/gin-gonic/gin/otelgin/option.go

```diff
@@ -23,8 +23,8 @@ type config struct {
 // be traced. A Filter must return true if the request should be traced.
 type Filter func(*http.Request) bool
 
-// SpanNameFormatter is used to set span name by http.request.
-type SpanNameFormatter func(r *http.Request) string
+// SpanNameFormatter is used to set span name by http.Request.
+type SpanNameFormatter func(routeName string, r *http.Request) string
```

Comment:
This changes the public API, and isn't what the PR describes. It should be its own PR.

Reply:
OK, I will create two PR for those changes.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5734 created for `http.route` change. @dmathieu